test/Microsoft.Azure.Devices.Edge.Test.Common/certs/Certificates.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Common.Certs
+{
+    public class Certificates
+    {
+        private string[] GetFileLocation(string deviceId)
+        {
+            return new[]
+            {
+                $"certs/iot-device-{deviceId}-full-chain.cert.pem",
+                $"private/iot-device-{deviceId}.key.pem"
+            };
+        }
+
+        public string CertificatePath { get; protected set; }
+
+        public string KeyPath { get; protected set; }
+
+        protected Certificates()
```

Comment:
No need for an empty constructor...

Reply:
An empty constructor is necessary for the code to compile for the derived `CaCertificate` class to set its params

Reply:
Let's try to find a way to fix this in the future...